avocado/core/job.py

```diff
@@ -75,7 +75,7 @@ def __init__(self, args=None):
         if args is None:
             args = argparse.Namespace()
         self.args = args
-        self.urls = getattr(args, "url", [])
+        self.references = getattr(args, "reference", [])
```

Comment:
In the context of tests, the reference is quite clear. But on first sight I have problems with matching `references` to tests in the context of `job`. Probably I'll learn that, but I think `self.test_references` would be much clearer.